org.adempiere.ui.zk/WEB-INF/src/org/adempiere/webui/factory/DefaultFeedbackService.java

```diff
@@ -113,10 +115,20 @@ protected void showEmailDialog(byte[] imageBytes) {
 			dialog.setAttribute(Window.MODE_KEY, Mode.OVERLAPPED);			
 			
 			MSystem system = MSystem.get(Env.getCtx());
-			if (!Util.isEmpty(system.getSupportEMail())) 
+
+			for (String s : getFeedbackRecipient(MSysConfig.FEEDBACK_EMAIL_TO).split(","))
+				if(EMail.validate(s.trim()))
+					dialog.addTo(s.trim(), Util.isEmpty(dialog.getTo()));
```

Comment:
if it is always append, the second argument should always be false

Reply:
Condition is true with the first recipient, then is false.
If I hardcode it to false, we can have only one recipient

Reply:
No, that only determine whether the email is add to front or append to end.